src/main/java/io/github/zerthick/protectionperms/ProtectionPerms.java

```diff
@@ -78,13 +78,18 @@ public void onServerInit(GameInitializationEvent event) {
                 logger.warn("Error loading default config! Error: " + e.getMessage());
             }
         }
-
         //Load invert
         try {
-            CommentedConfigurationNode linksNode = configLoader.load().getNode("invert");
-            boolean invert = linksNode.getBoolean(false);
-            PermHandler.getInstance().init(invert);
-
+            CommentedConfigurationNode modeNode = configLoader.load().getNode("mode");
+            if (modeNode.isVirtual()) {
+                logger.warn("Config corrupt! Using defaults (whitelist mode)\nDelete " + instance.getId() + ".conf to fix");
```

Comment:
We say that we are using the default mode "whitelist" here, but we never actually set it.

Reply:
`String node = modeNode.getString("whitelist");` sets the mode to whitelist, or gets the mode set in the config.

Reply:
Oops you're right, forgot about that default setting